app/views/videos/index/_videos_table.html.erb

```diff
@@ -2,7 +2,7 @@
   <thead>
     <tr>
       <th>Title</th>
-      <th>Category</th>
+      <th>Subject</th>
```

Comment:
We should be consistent in the way we reference these. If it's "Subject" to the user, it should be "Subject" to the developers too.